lsp-mode.el

```diff
@@ -1105,6 +1110,15 @@ to the beginning and ending points in the range correspondingly."
       (cons start end))
      (parent (lsp--find-wrapping-range parent)))))
 
+(defun lsp--focus-buffer (&rest args)
+  "Focus buffer handler.
+
+Remember the last active workspace set so that we can try that first
```

Comment:
typo: workspace -> workspaces.

Reply:
Ha. When I used the word "set" I meant in the discrete math sense of the word. I can see how this comment is confusing and will update it.

Reply:
Ah, it was actually me misreading the comment. It is fine to stay as it is.

Reply:
Fixed.